src/library/basetracktablemodel.cpp

```diff
@@ -795,6 +795,21 @@ QVariant BaseTrackTableModel::roleValue(
                 return QVariant();
             }
             return QVariant::fromValue(StarRating(rawValue.toInt()));
+        case ColumnCache::COLUMN_LIBRARYTABLE_REPLAYGAIN:
+            double rgRatio;
```

Comment:
Duplicating all this code for both Qt::EditRole and Qt::DisplayRole is a bad idea.

Reply:
added a anon-namespace func